site/src/uiuisms.rs

```diff
@@ -228,15 +228,15 @@ uiuisms!(
     /// Dot product
     "/+× [1 2 3] [4 ¯5 6]",
     /// Cross product
-    "1_2_3 4_5_6\n↻1-∩(×↻1)⊃:⊙∘",
+    "1_2_3 4_5_6\n↻2-∩(×↻2)⊙.,",
     /// Matrix product
     "[7_8_9 10_11_12] [1_2 3_4 5_6]\n⍜⍉⊞(/+×)",
     /// Matrix power (Also works with scalars)
     "4 [1_2 3_4]\n⊙◌⍥(⊞(/+×)⊙⍉,):⊞=.⇡⬚1⊢△,",
     /// Repeat a function and collect intermediate results into an array
     "[⍥(×2.)] 10 1",
     /// Pad an array with 0s
-    "[1_2_3 4_5_6]\n↻⊟.¯1⬚0↙+2△.",
+    "[1_2_3 4_5_6]\n↻∵⋅¯1:⬚0↙+2,⟜△",
```

Comment:
What's the rational for this one?

Reply:
It makes it work for arrays of any rank, whereas the original only worked for rank 2 arrays.

Reply:
If it's just making a list of `¯1`s with the same length as the shape, it should probably use `keep` instead.

Reply:
Right, I forgot `keep` could do that.

Reply:
Updated